app/ui-angular/src/app/store/integration/integration.service.ts

```diff
@@ -23,6 +23,9 @@ function getFlowsCount(integration: Integration) {
 }
 
 function transform(integration: Integration): Integration {
+  if (typeof(integration.flows) === 'undefined' || integration.flows === null) {
```

Comment:
`typeof integration.flows === 'undefined'` will cover both `null` and `undefined` for ya but it's just a nitpick :-)

Reply:
Is that a TypeScript thing? `typeof(null) === "object"` in JavaScript...

Reply:
`!integrations.flow` and call it a day?

Reply:
I was previously warned not to do those :smirk_cat:, thought I'd do whatever you folk think is the best practice, no worries :+1:

Reply:
@gashcrumb @riccardo-forina `!integration.flows` then?

Reply:
I'll actually be that guy and propose yet another option: `integration.flows = integration.flows || []`